sidetree/lib/core/MongoDbDidTypeStore.ts

```diff
@@ -0,0 +1,40 @@
+import DidTypeModel from './models/DidTypeModel';
+import IDidTypeStore from './interfaces/IDidTypeStore';
+import { Long } from 'mongodb';
+import MongoDbStore from '../common/MongoDbStore';
+
+/**
+ * Implementation of DidTypeStore that stores the didtype data in
+ * a MongoDB database.
+ */
+export default class MongoDbDidTypeStore extends MongoDbStore implements IDidTypeStore {
+  /** MongoDB collection name under the database where the operations are stored. */
+  public static readonly collectionName: string = 'didtype';
```

Comment:
following convention:

```suggestion
  public static readonly collectionName: string = 'did-type';
```